gcloud/pubsub/message.py

```diff
@@ -15,6 +15,11 @@
 """Define API Topics."""
 
 import base64
+import datetime
+
+import pytz
+
+RFC3369 = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
1. Did you mean `RFC3339`?
1. This should not be public.
1. Should we share this constant? It's also defined in `gcloud.storage.blob`.